### ISSUE TITLE
String-related instance of RemovedInDjango40Warning in Django 3.0

### ISSUE THREAD

**Issue by herrnici** [zostera/django-bootstrap4#183]
At least the following two deprecations exist:
```
bootstrap4/components.py:17: RemovedInDjango40Warning: django.utils.translation.ugettext() is deprecated in favor of django.utils.translation.gettext().
bootstrap4/text.py:11: RemovedInDjango40Warning: force_text() is deprecated in favor of force_str().
```

**Comment by herrnici** [comment]
Correction: Second deprecation was a duplicate of #173, was fixed in commit 4dd20b40aa56

**Comment by dyve** [comment]
Thanks. Should be fixed in upcoming release.

**Comment by dyve** [comment]
Fixed in v1.1.0
